system_tests/vision.py

```diff
@@ -478,13 +478,16 @@ def tearDown(self):
             value.delete()
 
     def _assert_color(self, color):
-        self.assertIsInstance(color.red, int)
-        self.assertIsInstance(color.green, int)
-        self.assertIsInstance(color.blue, int)
+        self.assertIsInstance(color.red, (int, float))
+        self.assertIsInstance(color.green, (int, float))
+        self.assertIsInstance(color.blue, (int, float))
+        if not isinstance(color.alpha, float):
+            self.assertIsInstance(color.alpha.value, float)
+        else:
+            self.assertIsInstance(color.alpha, float)
```

Comment:
Just the protobuf/gRPC backend. I can't find it now, but there was a little comment block in either the protobuf definition or the generated python code about why it's there. Basically to tell if it's been set or not...if I remember right.